src/inverseRobot.js

```diff
@@ -8,6 +8,27 @@
 
 function inverseRobot(robot) {
   // write code here
+  let tempKyesRobot = {};
+  let tempValueRobot = {};
+
+  tempKyesRobot = Object.keys(robot);
+  tempValueRobot = Object.values(robot);
+
+  for (let i = 0; i < tempValueRobot.length; i++) {
+    if (tempValueRobot.indexOf(tempValueRobot[i]) !== i) {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    delete robot[key];
+  }
+
+  for (const i in tempKyesRobot) {
+    robot[tempValueRobot[i]] = tempKyesRobot[i];
```

Comment:
This line of code is not clear. It's better to use meaningful variable names instead of 'i'. Also, consider using a single loop to check for duplicates and create the new object at the same time.